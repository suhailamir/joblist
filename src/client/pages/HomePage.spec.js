import React from 'react';
import { shallow, configure } from 'enzyme';
// setup file
import Adapter from 'enzyme-adapter-react-16';
import component from './HomePage';

configure({ adapter: new Adapter() });

describe('AldarContactCard', () => {
  let props;
  let getShallowComponent;

  beforeEach(() => {
    props = {};

    getShallowComponent = () => shallow(<component {...props} />, { context: {} });
  });

  describe('render', () => {
    it('should render with all props', () => {
      expect(getShallowComponent()).toMatchSnapshot();
    });
  });
});
