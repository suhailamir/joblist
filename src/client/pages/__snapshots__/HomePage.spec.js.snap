// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AldarContactCard render should render with all props 1`] = `ShallowWrapper {}`;

exports[`AldarContactCard render should render with all props 2`] = `ShallowWrapper {}`;
